### ISSUE TITLE
Summary - Missing files

### ISSUE THREAD

**Issue by Tzuridis** [AleksandraZhi/TeaCozy#4]
You have the files but there is nothing in them. Did you perhaps not push the changes you made locally to GitHub?

Ask the advisors for more help if you have git issues.

**Comment by AleksandraZhi** [comment]
You are absolutely right, I haven't realised it's not pushing automatically. I've updated it now.

**Comment by Tzuridis** [comment]
Close these old issues
